public/js/jquery.t.js

```diff
@@ -16,6 +16,17 @@
         });
     };
 
+    var setupResponsiveness = function ($queryDiv, $graphDiv, index, opts)  {
+        var debounced_draw = _.debounce(function () {
+            var shownHits = $queryDiv.find('.ghit > g').length;
+            $.graphIt($queryDiv, $graphDiv, shownHits, 0, opts);
```

Comment:
I think it should be `index` instead of `0`. Otherwise I see a lot of errors raised by d3 on resize in the console if 'View more' button is present in the graph. I think s/0/index intuitively makes more sense too.

To reproduce - blastn first 50 lines of a scaffold to that genome so you get a lot of hits.